test/servers/rpc-router.test.ts

```diff
@@ -0,0 +1,102 @@
+import 'mocha';
+import * as express from 'express';
+import * as sinon from 'sinon';
+import {RPCRouter, RPCServer} from '../../src/servers';
+import {expect} from '@loopback/testlab';
+
+describe('rpc-router', () => {
+  let router: RPCRouter;
+  let request: express.Request;
+  let response: express.Response;
+  // tslint:disable-next-line:no-any
+  let controller: any;
+
+  beforeEach(setupFakes);
+  it('routes to existing controller and method', async () => {
+    await router.routeRequest(request, response);
+    const stub = response.send as sinon.SinonStub;
+    expect(stub.called);
```

Comment:
Have you seen this assertion to fail? I believe it's a no-op:

```
> expect(false)
Assertion {
  obj: false,
  anyOne: false,
  negate: false,
  params: { actual: false } }
```

I think you wanted to say `expect(stub.called).to.be.true()` but please don't do that - the assertion failure will be rather unhelpful. Use [sinon assertions](http://sinonjs.org/releases/v1.17.6/assertions/) instead:

```ts
sinon.assert.called(stub);
```

We tried to integrate should with sinon via https://github.com/shouldjs/sinon, but it did not work :(

Reply:
Please fix other places using `expect(stub.called);` too